src/providers/realtime-provider.js

```diff
@@ -77,19 +89,44 @@ export default class RealtimeProvider {
     }
 
     subscribeToLimits(domainObject, callback) {
-        const subscriptionDetails = this.buildSubscriptionDetails(domainObject, callback);
-        const id = subscriptionDetails.subscriptionId;
-        this.subscriptionsById[id] = subscriptionDetails;
+        const qualifiedName = idToQualifiedName(domainObject.identifier.key);
+        this.observingLimitChanges[qualifiedName] = {
+            callback
+        };
+        //Only subscribe once when the first observer shows up. The following observers will use this subscription
+        if (Object.keys(this.observingLimitChanges).length === 1) {
+            if (this.mdbChangesUnsubscribe) {
+                this.mdbChangesUnsubscribe();
+            }
+            this.mdbChangesUnsubscribe = this.subscribeToMDBChanges();
```

Comment:
I'm in two minds about whether we need this subscription. We already have a subscription for mdb changes in the [object-provider](https://github.com/akhenry/openmct-yamcs/pull/366/files#diff-cdd6a3132b920b516a11c563087308e06e53c8f2ac4419c6dda004544da22658R212) which lives from load until Open MCT is closed. So, we could share that subscription and react to messages [here](https://github.com/akhenry/openmct-yamcs/pull/366/files#diff-58d0a7ee67c74185338cf7e69ad5f3ce1d68be3142fa93edd6f5f6af40f9c22eR314)